enzy_htp/geometry/sampling.py

```diff
@@ -344,7 +344,7 @@ def _serial_md_steps(
     results = []
     for i in range(parallel_runs):
         # create job path
-        sub_work_dir = f"{work_dir}/rep_{i}"
+        sub_work_dir = fs.get_valid_temp_name(f"{work_dir}/rep_{i}")
```

Comment:
Could we also do zero padding for `rep_{i}`? the change is only to `rep_{i:04d}'

Reply:
Changed!